TpTrace Verbose: 0 : 11700, 1, 2023/04/18, 15:15:11.600, 4229109707187, testhost.dll, Version: 17.3.2 Current process architecture: X64
TpTrace Verbose: 0 : 11700, 1, 2023/04/18, 15:15:11.602, 4229109734578, testhost.dll, Runtime location: C:\Program Files\dotnet\shared\Microsoft.NETCore.App\6.0.16
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:11.605, 4229109756912, testhost.dll, DefaultEngineInvoker.Invoke: Testhost process started with args :[--port, 62678],[--endpoint, 127.0.0.1:062678],[--role, client],[--parentprocessid, 15928],[--diag, C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\coverage-tool-output\diagnostics.host.23-04-18_15-15-11_35049_7.log],[--tracelevel, 4],[--datacollectionport, 62677],[--telemetryoptedin, false]
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:11.608, 4229109786815, testhost.dll, Setting up debug trace listener.
TpTrace Verbose: 0 : 11700, 1, 2023/04/18, 15:15:11.608, 4229109791585, testhost.dll, TestPlatformTraceListener.Setup: Replacing listener System.Diagnostics.DefaultTraceListener with TestHostTraceListener.
TpTrace Verbose: 0 : 11700, 1, 2023/04/18, 15:15:11.608, 4229109792252, testhost.dll, TestPlatformTraceListener.Setup: Added test platform trace listener.
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:11.609, 4229109799725, testhost.dll, DefaultEngineInvoker.SetParentProcessExitCallback: Monitoring parent process with id: '15928'
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:11.613, 4229109837135, testhost.dll, DefaultEngineInvoker.GetConnectionInfo: Initialize communication on endpoint address: '127.0.0.1:062678'
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:11.623, 4229109940168, testhost.dll, SocketClient.Start: connecting to server endpoint: 127.0.0.1:062678
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:11.630, 4229110015429, testhost.dll, DefaultEngineInvoker.ConnectToDatacollector: Connecting to datacollector, port: 62677
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:11.634, 4229110051465, testhost.dll, Trying to connect to server on socket : 127.0.0.1:62677 
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:11.634, 4229110053577, testhost.dll, SocketClient.OnServerConnected: connected to server endpoint: 127.0.0.1:062678
TpTrace Verbose: 0 : 11700, 1, 2023/04/18, 15:15:11.634, 4229110055129, testhost.dll, SocketCommunicationManager : SetupClientAsync : Attempting to connect to the server.
TpTrace Information: 0 : 11700, 10, 2023/04/18, 15:15:11.637, 4229110083929, testhost.dll, Connected to the server successfully 
TpTrace Information: 0 : 11700, 10, 2023/04/18, 15:15:11.637, 4229110085406, testhost.dll, Using the buffer size of 16384 bytes
TpTrace Verbose: 0 : 11700, 1, 2023/04/18, 15:15:11.638, 4229110087090, testhost.dll, SocketCommunicationManager.WaitForServerConnection took: 0 ms, with 90000 ms timeout, and finished with True.
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:11.638, 4229110088831, testhost.dll, DefaultEngineInvoker.Invoke: Start Request Processing.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.638, 4229110090779, testhost.dll, MulticastDelegateUtilities.SafeInvoke: SocketClient: ServerConnected: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.TestRequestHandler.<InitializeCommunication>b__31_0, took 0 ms.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.638, 4229110092115, testhost.dll, Connected to server, and starting MessageLoopAsync
TpTrace Information: 0 : 11700, 12, 2023/04/18, 15:15:11.639, 4229110099101, testhost.dll, DefaultEngineInvoker.StartProcessingAsync: Connected to vstest.console, Starting process requests.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.642, 4229110128385, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.642, 4229110132556, testhost.dll, TcpClientExtensions.MessageLoopAsync: NotifyDataAvailable remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:11.646, 4229110168435, testhost.dll, TestRequestHandler.OnMessageReceived: received message: (ProtocolVersion) -> {"MessageType":"ProtocolVersion","Payload":7}
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.796, 4229111672704, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestSession.Message","Payload":{"MessageLevel":0,"Message":"Logging TestHost Diagnostics in file: C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\coverage-tool-output\\diagnostics.host.23-04-18_15-15-11_35049_7.log"}}
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.796, 4229111676203, testhost.dll, MulticastDelegateUtilities.SafeInvoke: LengthPrefixCommunicationChannel: MessageReceived: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.TestRequestHandler.OnMessageReceived, took 153 ms.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.797, 4229111677388, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.797, 4229111678819, testhost.dll, TcpClientExtensions.MessageLoopAsync: NotifyDataAvailable remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:11.797, 4229111680686, testhost.dll, TestRequestHandler.OnMessageReceived: received message: (TestExecution.Initialize) -> {"Version":7,"MessageType":"TestExecution.Initialize","Payload":["C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\NUnit3.TestAdapter.dll"]}
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.809, 4229111804651, testhost.dll, MulticastDelegateUtilities.SafeInvoke: LengthPrefixCommunicationChannel: MessageReceived: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.TestRequestHandler.OnMessageReceived, took 12 ms.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.810, 4229111806523, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.810, 4229111807781, testhost.dll, TcpClientExtensions.MessageLoopAsync: NotifyDataAvailable remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:11.810, 4229111809680, testhost.dll, TestRequestHandler.OnMessageReceived: received message: (TestExecution.StartWithSources) -> {"Version":7,"MessageType":"TestExecution.StartWithSources","Payload":{"AdapterSourceMap":{"_none_":["C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll"]},"RunSettings":"<RunSettings>\r\n  <RunConfiguration>\r\n    <ResultsDirectory>C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\coverage-tool-output</ResultsDirectory>\r\n    <TargetFrameworkVersion>.NETCoreApp,Version=v6.0</TargetFrameworkVersion>\r\n    <DesignMode>False</DesignMode>\r\n    <CollectSourceInformation>False</CollectSourceInformation>\r\n  </RunConfiguration>\r\n  <LoggerRunSettings>\r\n    <Loggers>\r\n      <Logger friendlyName=\"blame\" enabled=\"True\" />\r\n      <Logger friendlyName=\"Console\" uri=\"logger://microsoft/TestPlatform/ConsoleLogger/v1\" assemblyQualifiedName=\"Microsoft.VisualStudio.TestPlatform.CommandLine.Internal.ConsoleLogger, vstest.console, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a\" codeBase=\"C:\\Program Files\\dotnet\\sdk\\7.0.202\\vstest.console.dll\" enabled=\"True\" />\r\n    </Loggers>\r\n  </LoggerRunSettings>\r\n  <DataCollectionRunSettings>\r\n    <DataCollectors>\r\n      <DataCollector friendlyName=\"blame\" enabled=\"True\">\r\n        <Configuration>\r\n          <ResultsDirectory>C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\coverage-tool-output</ResultsDirectory>\r\n        </Configuration>\r\n      </DataCollector>\r\n    </DataCollectors>\r\n  </DataCollectionRunSettings>\r\n</RunSettings>","TestExecutionContext":{"FrequencyOfRunStatsChangeEvent":10,"RunStatsChangeEventTimeout":"00:00:01.5000000","InIsolation":false,"KeepAlive":false,"AreTestCaseLevelEventsRequired":false,"IsDebug":false,"TestCaseFilter":null,"FilterOptions":null},"Package":null}}
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.810, 4229111811430, testhost.dll, TestRequestHandler.OnMessageReceived: Running job 'TestExecution.Initialize'.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.813, 4229111842320, testhost.dll, TestExecutorService: Loading the extensions
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.816, 4229111867858, testhost.dll, TestPluginCache.DiscoverTestExtensions: finding test extensions in assemblies ends with: TestAdapter.dll TPluginInfo: Microsoft.VisualStudio.TestPlatform.Common.ExtensionFramework.Utilities.TestExecutorPluginInformation TExtension: Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter.ITestExecutor
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.817, 4229111880803, testhost.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.817, 4229111885069, testhost.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.817, 4229111886088, testhost.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.819, 4229111900430, testhost.dll, AssemblyResolver.ctor: Creating AssemblyResolver with searchDirectories C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.821, 4229111917492, testhost.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using extension path.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.821, 4229111919473, testhost.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.821, 4229111920222, testhost.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.821, 4229111920809, testhost.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.821, 4229111921671, testhost.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using allExtensionPaths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.822, 4229111933491, testhost.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.828, 4229111990581, testhost.dll, AssemblyResolver.OnResolve: NUnit3.TestAdapter: Resolving assembly.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.828, 4229111992042, testhost.dll, AssemblyResolver.OnResolve: NUnit3.TestAdapter: Searching in: 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output'.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.829, 4229112003381, testhost.dll, AssemblyResolver.OnResolve: NUnit3.TestAdapter: Loading assembly 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll'.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.830, 4229112010711, testhost.dll, AssemblyResolver.OnResolve: Resolved assembly: NUnit3.TestAdapter, from path: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.847, 4229112180633, testhost.dll, MulticastDelegateUtilities.SafeInvoke: LengthPrefixCommunicationChannel: MessageReceived: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.TestRequestHandler.OnMessageReceived, took 37 ms.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:11.847, 4229112182003, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.862, 4229112327350, testhost.dll, MetadataReaderExtensionsHelper: Discovering extensions inside assembly 'NUnit3.TestAdapter, Version=4.3.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac' file path 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll'
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.869, 4229112405576, testhost.dll, AssemblyResolver.OnResolve: nunit.engine.api: Resolving assembly.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.870, 4229112406831, testhost.dll, AssemblyResolver.OnResolve: nunit.engine.api: Searching in: 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output'.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.870, 4229112410570, testhost.dll, AssemblyResolver.OnResolve: nunit.engine.api: Loading assembly 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\nunit.engine.api.dll'.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.870, 4229112414539, testhost.dll, AssemblyResolver.OnResolve: Resolved assembly: nunit.engine.api, from path: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\nunit.engine.api.dll
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.874, 4229112450289, testhost.dll, GetTestExtensionFromType: Register extension with identifier data 'executor://NUnit3TestExecutor' and type 'NUnit.VisualStudio.TestAdapter.NUnit3TestExecutor, NUnit3.TestAdapter, Version=4.3.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac' inside file 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll'
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.877, 4229112480466, testhost.dll, TestPluginCache: Discovered the extensions using extension path ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.877, 4229112485610, testhost.dll, TestPluginCache: Discoverers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.878, 4229112486484, testhost.dll, TestPluginCache: Executors are 'executor://NUnit3TestExecutor'.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.878, 4229112487244, testhost.dll, TestPluginCache: Executors2 are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.878, 4229112487977, testhost.dll, TestPluginCache: Setting providers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.878, 4229112488683, testhost.dll, TestPluginCache: Loggers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.878, 4229112489488, testhost.dll, TestPluginCache: TestHosts are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.878, 4229112490171, testhost.dll, TestPluginCache: DataCollectors are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.880, 4229112515329, testhost.dll, TestPluginCache.DiscoverTestExtensions: finding test extensions in assemblies ends with: TestAdapter.dll TPluginInfo: Microsoft.VisualStudio.TestPlatform.Common.ExtensionFramework.Utilities.TestExecutorPluginInformation2 TExtension: Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter.ITestExecutor2
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112518262, testhost.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112518955, testhost.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112519418, testhost.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112521863, testhost.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112522465, testhost.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using extension path.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112522922, testhost.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112523352, testhost.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112523771, testhost.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112524205, testhost.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using allExtensionPaths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112524752, testhost.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.881, 4229112525625, testhost.dll, MetadataReaderExtensionsHelper: Discovering extensions inside assembly 'NUnit3.TestAdapter, Version=4.3.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac' file path 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll'
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.882, 4229112532404, testhost.dll, TestPluginCache: Discovered the extensions using extension path ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.882, 4229112533091, testhost.dll, TestPluginCache: Discoverers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.882, 4229112533615, testhost.dll, TestPluginCache: Executors are 'executor://NUnit3TestExecutor'.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.882, 4229112534449, testhost.dll, TestPluginCache: Executors2 are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.882, 4229112534905, testhost.dll, TestPluginCache: Setting providers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.882, 4229112535312, testhost.dll, TestPluginCache: Loggers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.882, 4229112535747, testhost.dll, TestPluginCache: TestHosts are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.882, 4229112536206, testhost.dll, TestPluginCache: DataCollectors are ''.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.887, 4229112579056, testhost.dll, TestPluginManager.CreateTestExtension: Attempting to load test extension: NUnit.VisualStudio.TestAdapter.NUnit3TestExecutor
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.887, 4229112583276, testhost.dll, TestExecutorExtensionManager: Loading executor NUnit.VisualStudio.TestAdapter.NUnit3TestExecutor
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.887, 4229112583920, testhost.dll, TestExecutorService: Loaded the executors
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.889, 4229112600632, testhost.dll, TestPluginCache.DiscoverTestExtensions: finding test extensions in assemblies ends with: TestAdapter.dll TPluginInfo: Microsoft.VisualStudio.TestPlatform.Common.ExtensionFramework.Utilities.TestSettingsProviderPluginInformation TExtension: Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter.ISettingsProvider
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.890, 4229112607966, testhost.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.890, 4229112610422, testhost.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.890, 4229112612443, testhost.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.890, 4229112616309, testhost.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.891, 4229112619052, testhost.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using extension path.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.891, 4229112619701, testhost.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.891, 4229112620925, testhost.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.891, 4229112621895, testhost.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.891, 4229112622484, testhost.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using allExtensionPaths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.891, 4229112623621, testhost.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.891, 4229112624849, testhost.dll, MetadataReaderExtensionsHelper: Discovering extensions inside assembly 'NUnit3.TestAdapter, Version=4.3.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac' file path 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll'
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.893, 4229112637484, testhost.dll, TestPluginCache: Discovered the extensions using extension path ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.893, 4229112638644, testhost.dll, TestPluginCache: Discoverers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.893, 4229112639191, testhost.dll, TestPluginCache: Executors are 'executor://NUnit3TestExecutor'.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.893, 4229112639688, testhost.dll, TestPluginCache: Executors2 are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.893, 4229112640361, testhost.dll, TestPluginCache: Setting providers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.893, 4229112640882, testhost.dll, TestPluginCache: Loggers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.893, 4229112641360, testhost.dll, TestPluginCache: TestHosts are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.893, 4229112641871, testhost.dll, TestPluginCache: DataCollectors are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.894, 4229112652699, testhost.dll, TestExecutorService: Loaded the settings providers
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.894, 4229112653918, testhost.dll, TestExecutorService: Loaded the extensions
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.895, 4229112659613, testhost.dll, TestRequestHandler.OnMessageReceived: Running job 'TestExecution.StartWithSources'.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.981, 4229113517042, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseSessionStart: Invoking callbacks was skipped because there are no subscribers.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.985, 4229113556586, testhost.dll, TestDiscoveryManager: Discovering tests from sources C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\DDT Sumator.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.987, 4229113584376, testhost.dll, TestPluginCache.DiscoverTestExtensions: finding test extensions in assemblies ends with: TestAdapter.dll TPluginInfo: Microsoft.VisualStudio.TestPlatform.Common.ExtensionFramework.Utilities.TestDiscovererPluginInformation TExtension: Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter.ITestDiscoverer
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.987, 4229113585885, testhost.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113586441, testhost.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113586911, testhost.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113588657, testhost.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113589220, testhost.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using extension path.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113589688, testhost.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113590127, testhost.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113590584, testhost.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113591048, testhost.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using allExtensionPaths: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113591602, testhost.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.988, 4229113592534, testhost.dll, MetadataReaderExtensionsHelper: Discovering extensions inside assembly 'NUnit3.TestAdapter, Version=4.3.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac' file path 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll'
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.991, 4229113617134, testhost.dll, GetTestExtensionFromType: Register extension with identifier data 'NUnit.VisualStudio.TestAdapter.NUnit3TestDiscoverer, NUnit3.TestAdapter, Version=4.3.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac' and type 'NUnit.VisualStudio.TestAdapter.NUnit3TestDiscoverer, NUnit3.TestAdapter, Version=4.3.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac' inside file 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\NUnit3.TestAdapter.dll'
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.991, 4229113619107, testhost.dll, TestPluginCache: Discovered the extensions using extension path ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.991, 4229113619826, testhost.dll, TestPluginCache: Discoverers are 'NUnit.VisualStudio.TestAdapter.NUnit3TestDiscoverer, NUnit3.TestAdapter, Version=4.3.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac'.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.991, 4229113620348, testhost.dll, TestPluginCache: Executors are 'executor://NUnit3TestExecutor'.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.991, 4229113620823, testhost.dll, TestPluginCache: Executors2 are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.991, 4229113621279, testhost.dll, TestPluginCache: Setting providers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.991, 4229113621716, testhost.dll, TestPluginCache: Loggers are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.991, 4229113622131, testhost.dll, TestPluginCache: TestHosts are ''.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:11.991, 4229113622530, testhost.dll, TestPluginCache: DataCollectors are ''.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:11.995, 4229113658974, testhost.dll, PEReaderHelper.GetAssemblyType: Determined assemblyType:'Managed' for source: 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\DDT Sumator.dll'
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:12.000, 4229113715841, testhost.dll, BaseRunTests.RunTestInternalWithExecutors: Running tests for executor://NUnit3TestExecutor
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.003, 4229113741420, testhost.dll, AssemblyResolver.OnResolve: nunit.engine.core: Resolving assembly.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.003, 4229113742555, testhost.dll, AssemblyResolver.OnResolve: nunit.engine.core: Searching in: 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output'.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.003, 4229113745970, testhost.dll, AssemblyResolver.OnResolve: nunit.engine.core: Loading assembly 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\nunit.engine.core.dll'.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.004, 4229113750328, testhost.dll, AssemblyResolver.OnResolve: Resolved assembly: nunit.engine.core, from path: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\nunit.engine.core.dll
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.013, 4229113840546, testhost.dll, NUnit Adapter 4.3.0.0: Test execution started
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:12.013, 4229113842327, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestSession.Message","Payload":{"MessageLevel":0,"Message":"NUnit Adapter 4.3.0.0: Test execution started"}}
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:12.014, 4229113846778, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestRunMessageLoggerProxy.SendMessage: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Execution.RunTestsWithSources.OnTestRunMessage, took 0 ms.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.053, 4229114242416, testhost.dll, Running all tests in C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\DDT Sumator.dll
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:12.053, 4229114244257, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestSession.Message","Payload":{"MessageLevel":0,"Message":"Running all tests in C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll"}}
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:12.054, 4229114246442, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestRunMessageLoggerProxy.SendMessage: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Execution.RunTestsWithSources.OnTestRunMessage, took 0 ms.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.062, 4229114330151, testhost.dll, AssemblyResolver.OnResolve: testcentric.engine.metadata: Resolving assembly.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.062, 4229114331487, testhost.dll, AssemblyResolver.OnResolve: testcentric.engine.metadata: Searching in: 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output'.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.062, 4229114335627, testhost.dll, AssemblyResolver.OnResolve: testcentric.engine.metadata: Loading assembly 'C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\testcentric.engine.metadata.dll'.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.063, 4229114340625, testhost.dll, AssemblyResolver.OnResolve: Resolved assembly: testcentric.engine.metadata, from path: C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\bin\Debug\net6.0\fine-code-coverage\build-output\testcentric.engine.metadata.dll
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:12.613, 4229119844828, testhost.dll,    NUnit3TestExecutor discovered 15 of 15 NUnit test cases using Current Discovery mode, Non-Explicit run
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:12.614, 4229119846576, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestSession.Message","Payload":{"MessageLevel":0,"Message":"   NUnit3TestExecutor discovered 15 of 15 NUnit test cases using Current Discovery mode, Non-Explicit run"}}
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:12.614, 4229119849063, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestRunMessageLoggerProxy.SendMessage: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Execution.RunTestsWithSources.OnTestRunMessage, took 0 ms.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:12.852, 4229122230686, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:13.863, 4229132340559, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:14.873, 4229142442135, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:15.883, 4229152543579, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:16.897, 4229162679140, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:17.905, 4229172764371, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:18.912, 4229182829894, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:19.917, 4229192883120, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:20.930, 4229203016197, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:21.936, 4229213073324, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:22.940, 4229223115158, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:23.945, 4229233166264, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.339, 4229237097363, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","+","3","Result: 8") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.339, 4229237101991, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.346, 4229237171346, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","+","3","Result: 8").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.348, 4229237190482, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 1 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.351, 4229237217066, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.351, 4229237218858, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5","+","3","Result: 8")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.351, 4229237224831, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","-","3","Result: 2") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.351, 4229237225609, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237226883, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","-","3","Result: 2").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237227712, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237228287, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237228836, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5","-","3","Result: 2")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237231520, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","/","3","Result: 1.66666666667") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237232105, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237232730, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","/","3","Result: 1.66666666667").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237233785, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237234409, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.352, 4229237234965, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5","/","3","Result: 1.66666666667")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237237857, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","*","3","Result: 15") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237238446, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237239155, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","*","3","Result: 15").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237239751, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237240244, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237240696, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5","*","3","Result: 15")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237243165, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5.5","+","2","Result: 7.5") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237243666, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237244226, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5.5","+","2","Result: 7.5").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237244739, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237245292, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.353, 4229237245715, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5.5","+","2","Result: 7.5")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.354, 4229237248051, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5.5","-","2","Result: 3.5") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.354, 4229237248596, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.354, 4229237249224, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5.5","-","2","Result: 3.5").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.354, 4229237249740, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.354, 4229237250195, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.354, 4229237250609, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5.5","-","2","Result: 3.5")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237287895, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5.5","/","2","Result: 2.75") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237289228, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237290041, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5.5","/","2","Result: 2.75").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237290753, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237291297, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237291756, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5.5","/","2","Result: 2.75")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237294285, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5.5","*","2","Result: 11") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237294792, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237295328, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5.5","*","2","Result: 11").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237295855, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.358, 4229237296301, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237296712, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5.5","*","2","Result: 11")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237298689, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("1.5e53","*","150","Result: 2.25e+55") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237299186, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237299737, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("1.5e53","*","150","Result: 2.25e+55").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237300328, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237300825, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237301246, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("1.5e53","*","150","Result: 2.25e+55")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237303183, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("1.5e53","/","150","Result: 1e+51") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237303675, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237304221, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("1.5e53","/","150","Result: 1e+51").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237304729, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237305178, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.359, 4229237305589, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("1.5e53","/","150","Result: 1e+51")' in inProgress list.
TpTrace Information: 0 : 11700, 14, 2023/04/18, 15:15:24.362, 4229237326700, testhost.dll, Sending test run statistics
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.392, 4229237631705, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestExecution.StatsChange","Payload":{"NewTestResults":[{"TestCase":{"Id":"508c23c4-b7df-845f-19a8-61e6f6638c21","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5\",\"+\",\"3\",\"Result: 8\")","DisplayName":"TestCalculatorWebApp(\"5\",\"+\",\"3\",\"Result: 8\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"217462821"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5\",\"+\",\"3\",\"Result: 8\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6843120","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3256115+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"217462821"}]},{"TestCase":{"Id":"76a6786c-f49e-11e1-cdbc-076302f8046e","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5\",\"-\",\"3\",\"Result: 2\")","DisplayName":"TestCalculatorWebApp(\"5\",\"-\",\"3\",\"Result: 2\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"772533455"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5\",\"-\",\"3\",\"Result: 2\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846560","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3259558+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"772533455"}]},{"TestCase":{"Id":"46dcd849-e447-a46d-3366-2e7eb6b6e16d","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5\",\"/\",\"3\",\"Result: 1.66666666667\")","DisplayName":"TestCalculatorWebApp(\"5\",\"/\",\"3\",\"Result: 1.66666666667\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1830907687"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5\",\"/\",\"3\",\"Result: 1.66666666667\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846710","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3259706+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1830907687"}]},{"TestCase":{"Id":"6f302df4-58c7-a7af-c37e-3fe9f7b8a26c","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5\",\"*\",\"3\",\"Result: 15\")","DisplayName":"TestCalculatorWebApp(\"5\",\"*\",\"3\",\"Result: 15\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"999986340"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5\",\"*\",\"3\",\"Result: 15\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846740","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3259738+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"999986340"}]},{"TestCase":{"Id":"f18fa199-5080-f9e1-0466-4a483ba7f5d3","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5.5\",\"+\",\"2\",\"Result: 7.5\")","DisplayName":"TestCalculatorWebApp(\"5.5\",\"+\",\"2\",\"Result: 7.5\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"412921174"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5.5\",\"+\",\"2\",\"Result: 7.5\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846770","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.325977+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"412921174"}]},{"TestCase":{"Id":"f71c996d-781f-7c54-0852-2a9a239f38a8","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5.5\",\"-\",\"2\",\"Result: 3.5\")","DisplayName":"TestCalculatorWebApp(\"5.5\",\"-\",\"2\",\"Result: 3.5\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1176263128"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5.5\",\"-\",\"2\",\"Result: 3.5\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846800","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3259801+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1176263128"}]},{"TestCase":{"Id":"35c6d32b-8397-992e-dc4b-83dd9288bb54","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5.5\",\"/\",\"2\",\"Result: 2.75\")","DisplayName":"TestCalculatorWebApp(\"5.5\",\"/\",\"2\",\"Result: 2.75\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"161147354"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5.5\",\"/\",\"2\",\"Result: 2.75\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846830","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.325983+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"161147354"}]},{"TestCase":{"Id":"1510be4f-3c56-8719-e50a-72f3277b1821","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5.5\",\"*\",\"2\",\"Result: 11\")","DisplayName":"TestCalculatorWebApp(\"5.5\",\"*\",\"2\",\"Result: 11\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"583998339"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5.5\",\"*\",\"2\",\"Result: 11\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846870","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3259864+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"583998339"}]},{"TestCase":{"Id":"861e0094-2c93-f10b-1466-f33889a5a640","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"1.5e53\",\"*\",\"150\",\"Result: 2.25e+55\")","DisplayName":"TestCalculatorWebApp(\"1.5e53\",\"*\",\"150\",\"Result: 2.25e+55\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1147229931"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"1.5e53\",\"*\",\"150\",\"Result: 2.25e+55\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846900","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3259895+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1147229931"}]},{"TestCase":{"Id":"998eb392-41a2-a0b1-8b41-aa7023f05063","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"1.5e53\",\"/\",\"150\",\"Result: 1e+51\")","DisplayName":"TestCalculatorWebApp(\"1.5e53\",\"/\",\"150\",\"Result: 1e+51\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1571175794"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"1.5e53\",\"/\",\"150\",\"Result: 1e+51\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846930","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.325993+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1571175794"}]}],"TestRunStatistics":{"ExecutedTests":10,"Stats":{"Failed":10}},"ActiveTests":[]}}
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.393, 4229237638265, testhost.dll, TestRunCache: OnNewTestResult: Notified the onCacheHit callback.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.393, 4229237642043, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("Infinity","+","Infinity","Result: Infinity") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.393, 4229237642790, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.393, 4229237643635, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("Infinity","+","Infinity","Result: Infinity").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.393, 4229237644369, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.393, 4229237644940, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.393, 4229237645436, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("Infinity","+","Infinity","Result: Infinity")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237648174, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","*","Result: invalid input","Result: invalid input") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237648862, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237649593, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","*","Result: invalid input","Result: invalid input").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237650282, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237650839, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237651320, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5","*","Result: invalid input","Result: invalid input")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237653850, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","/","Result: invalid input","Result: invalid input") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237654441, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237655105, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","/","Result: invalid input","Result: invalid input").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237655739, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.394, 4229237656298, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237656849, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5","/","Result: invalid input","Result: invalid input")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237659195, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","+","Result: invalid input","Result: invalid input") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237659773, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237660428, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","+","Result: invalid input","Result: invalid input").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237661102, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237661681, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237662177, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5","+","Result: invalid input","Result: invalid input")' in inProgress list.
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237664475, testhost.dll, TestExecutionRecorder.RecordEnd: test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","-","Result: invalid input","Result: invalid input") execution completed.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237665039, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237665690, testhost.dll, TestExecutionRecorder.RecordResult: Received result for test: DDT_Sumator.SeleniumTests.TestCalculatorWebApp("5","-","Result: invalid input","Result: invalid input").
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:24.395, 4229237666312, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseTestCaseEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerSendTestResult, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.396, 4229237666850, testhost.dll, TestRunCache: InProgressTests is null
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:24.396, 4229237667328, testhost.dll, TestRunCache: No test found corresponding to testResult 'TestCalculatorWebApp("5","-","Result: invalid input","Result: invalid input")' in inProgress list.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:24.946, 4229243169161, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Information: 0 : 11700, 10, 2023/04/18, 15:15:25.905, 4229252762715, testhost.dll, Sending test run statistics
TpTrace Verbose: 0 : 11700, 10, 2023/04/18, 15:15:25.906, 4229252772288, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestExecution.StatsChange","Payload":{"NewTestResults":[{"TestCase":{"Id":"c695ca74-39cf-266a-d90d-c919289ca477","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"Infinity\",\"+\",\"Infinity\",\"Result: Infinity\")","DisplayName":"TestCalculatorWebApp(\"Infinity\",\"+\",\"Infinity\",\"Result: Infinity\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1880184047"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"Infinity\",\"+\",\"Infinity\",\"Result: Infinity\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846960","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.325996+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1880184047"}]},{"TestCase":{"Id":"701c4edc-3197-0dba-71a4-afa24428a839","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5\",\"*\",\"Result: invalid input\",\"Result: invalid input\")","DisplayName":"TestCalculatorWebApp(\"5\",\"*\",\"Result: invalid input\",\"Result: invalid input\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"621023338"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5\",\"*\",\"Result: invalid input\",\"Result: invalid input\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6846990","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3259988+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"621023338"}]},{"TestCase":{"Id":"b68d9633-b23c-b0da-6387-0a92168cfb27","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5\",\"/\",\"Result: invalid input\",\"Result: invalid input\")","DisplayName":"TestCalculatorWebApp(\"5\",\"/\",\"Result: invalid input\",\"Result: invalid input\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"156747149"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5\",\"/\",\"Result: invalid input\",\"Result: invalid input\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6847020","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3260016+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"156747149"}]},{"TestCase":{"Id":"85fa094f-b932-dbc6-5b19-9423406e1c3f","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5\",\"+\",\"Result: invalid input\",\"Result: invalid input\")","DisplayName":"TestCalculatorWebApp(\"5\",\"+\",\"Result: invalid input\",\"Result: invalid input\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"2045523582"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5\",\"+\",\"Result: invalid input\",\"Result: invalid input\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6847060","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3260055+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"2045523582"}]},{"TestCase":{"Id":"15216b2a-d742-47ed-180f-817256cdc9fe","FullyQualifiedName":"DDT_Sumator.SeleniumTests.TestCalculatorWebApp(\"5\",\"-\",\"Result: invalid input\",\"Result: invalid input\")","DisplayName":"TestCalculatorWebApp(\"5\",\"-\",\"Result: invalid input\",\"Result: invalid input\")","ExecutorUri":"executor://NUnit3TestExecutor","Source":"C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\bin\\Debug\\net6.0\\fine-code-coverage\\build-output\\DDT Sumator.dll","CodeFilePath":null,"LineNumber":0,"Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1306358506"}]},"Attachments":[],"Outcome":2,"ErrorMessage":"OneTimeSetUp: OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.","ErrorStackTrace":"\n","DisplayName":"TestCalculatorWebApp(\"5\",\"-\",\"Result: invalid input\",\"Result: invalid input\")","Messages":[],"ComputerName":"RADOSTIN03","Duration":"00:00:11.6847090","StartTime":"2023-04-18T12:15:12.6412995+00:00","EndTime":"2023-04-18T12:15:24.3260091+00:00","Properties":[{"Key":{"Id":"NUnit.Seed","Label":"Seed","Category":"","Description":"","Attributes":0,"ValueType":"System.String"},"Value":"1306358506"}]}],"TestRunStatistics":{"ExecutedTests":15,"Stats":{"Failed":15}},"ActiveTests":[]}}
TpTrace Verbose: 0 : 11700, 10, 2023/04/18, 15:15:25.907, 4229252776998, testhost.dll, TestRunCache: OnNewTestResult: Notified the onCacheHit callback.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:25.952, 4229253226863, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:26.957, 4229263280509, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:27.963, 4229273336474, testhost.dll, TcpClientExtensions.MessageLoopAsync: Polling on remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:28.416, 4229277876226, testhost.dll, Setup failed for test fixture DDT_Sumator.SeleniumTests
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:28.417, 4229277877572, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestSession.Message","Payload":{"MessageLevel":1,"Message":"Setup failed for test fixture DDT_Sumator.SeleniumTests"}}
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:28.417, 4229277880276, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestRunMessageLoggerProxy.SendMessage: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Execution.RunTestsWithSources.OnTestRunMessage, took 0 ms.
TpTrace Warning: 0 : 11700, 14, 2023/04/18, 15:15:28.417, 4229277883476, testhost.dll, OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.
  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.
  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.
StackTrace:    at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)
   at OpenQA.Selenium.Remote.DriverServiceCommandExecutor.Execute(Command commandToExecute)
   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.WebDriver.set_Url(String value)
   at OpenQA.Selenium.Navigator.GoToUrl(String url)
   at DDT_Sumator.SeleniumTests.Setup() in C:\Users\raddk\source\repos\Team-Bandits\SeleniumExersice1\DDT Sumator\DDT_Sumator.cs:line 23
--HttpRequestException
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)
   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)
--IOException
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource<System.Int32>.GetResult(Int16 token)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
--SocketException

TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:28.417, 4229277885824, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestSession.Message","Payload":{"MessageLevel":1,"Message":"OpenQA.Selenium.WebDriverException : An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:62683/session/c7cd3f542f3a82adad29380d5b4cb2d1/url. The exception message was: An error occurred while sending the request.\r\n  ----> System.Net.Http.HttpRequestException : An error occurred while sending the request.\r\n  ----> System.IO.IOException : Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..\r\n  ----> System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host.\nStackTrace:    at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\r\n   at OpenQA.Selenium.Remote.DriverServiceCommandExecutor.Execute(Command commandToExecute)\r\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\r\n   at OpenQA.Selenium.WebDriver.set_Url(String value)\r\n   at OpenQA.Selenium.Navigator.GoToUrl(String url)\r\n   at DDT_Sumator.SeleniumTests.Setup() in C:\\Users\\raddk\\source\\repos\\Team-Bandits\\SeleniumExersice1\\DDT Sumator\\DDT_Sumator.cs:line 23\r\n--HttpRequestException\r\n   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\r\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\r\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\r\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\r\n--IOException\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\r\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource<System.Int32>.GetResult(Int16 token)\r\n   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\r\n--SocketException\r\n"}}
TpTrace Verbose: 0 : 11700, 14, 2023/04/18, 15:15:28.418, 4229277889029, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestRunMessageLoggerProxy.SendMessage: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Execution.RunTestsWithSources.OnTestRunMessage, took 0 ms.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:28.426, 4229277966678, testhost.dll, NUnit Adapter 4.3.0.0: Test execution complete
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:28.426, 4229277968114, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestSession.Message","Payload":{"MessageLevel":0,"Message":"NUnit Adapter 4.3.0.0: Test execution complete"}}
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:28.426, 4229277969881, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestRunMessageLoggerProxy.SendMessage: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Execution.RunTestsWithSources.OnTestRunMessage, took 0 ms.
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:28.431, 4229278024513, testhost.dll, BaseRunTests.RunTestInternalWithExecutors: Completed running tests for executor://NUnit3TestExecutor
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:28.435, 4229278065094, testhost.dll, MulticastDelegateUtilities.SafeInvoke: TestCaseEventsHandler.RaiseSessionEnd: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection.ProxyOutOfProcDataCollectionManager.TriggerTestSessionEnd, took 3 ms.
TpTrace Information: 0 : 11700, 4, 2023/04/18, 15:15:28.438, 4229278091097, testhost.dll, Sending test run complete
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:28.447, 4229278176799, testhost.dll, TestRequestHandler.SendData: sending data from testhost: {"Version":7,"MessageType":"TestExecution.Completed","Payload":{"TestRunCompleteArgs":{"TestRunStatistics":{"ExecutedTests":15,"Stats":{"Failed":15}},"IsCanceled":false,"IsAborted":false,"Error":null,"AttachmentSets":[],"InvokedDataCollectors":[],"ElapsedTimeInRunningTests":"00:00:16.4349352","Metrics":{},"DiscoveredExtensions":{"TestDiscoverers":["NUnit.VisualStudio.TestAdapter.NUnit3TestDiscoverer, NUnit3.TestAdapter, Version=4.3.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac"],"TestExecutors":["executor://NUnit3TestExecutor"],"TestExecutors2":[],"TestSettingsProviders":[]}},"LastRunTests":{"NewTestResults":[],"TestRunStatistics":{"ExecutedTests":15,"Stats":{"Failed":15}},"ActiveTests":[]},"RunAttachments":[],"ExecutorUris":["executor://nunit3testexecutor/"]}}
TpTrace Verbose: 0 : 11700, 4, 2023/04/18, 15:15:28.447, 4229278180492, testhost.dll, BaseRunTests.RunTests: Run is complete.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:28.447, 4229278183895, testhost.dll, TcpClientExtensions.MessageLoopAsync: NotifyDataAvailable remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:28.447, 4229278184937, testhost.dll, TestRequestHandler.OnMessageReceived: received message: (TestSession.Terminate) -> {"MessageType":"TestSession.Terminate","Payload":null}
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:28.447, 4229278185424, testhost.dll, Session End message received from server. Closing the connection.
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:28.448, 4229278189812, testhost.dll, Closing the connection!
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:28.448, 4229278191030, testhost.dll, SocketClient.Stop: Stop communication from server endpoint: 127.0.0.1:062678
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:28.448, 4229278191220, testhost.dll, SocketClient.Stop: Stop communication from server endpoint: 127.0.0.1:062678
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:28.448, 4229278191626, testhost.dll, SocketClient: Stop: Cancellation requested. Stopping message loop.
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:28.448, 4229278192124, testhost.dll, SocketClient: Stop: Cancellation requested. Stopping message loop.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:28.448, 4229278193660, testhost.dll, LengthPrefixCommunicationChannel.Dispose: Dispose reader and writer.
TpTrace Verbose: 0 : 11700, 1, 2023/04/18, 15:15:28.448, 4229278193834, testhost.dll, LengthPrefixCommunicationChannel.Dispose: Dispose reader and writer.
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:28.448, 4229278194233, testhost.dll, Closing the connection !
TpTrace Information: 0 : 11700, 1, 2023/04/18, 15:15:28.448, 4229278195581, testhost.dll, Testhost process exiting.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:28.448, 4229278195613, testhost.dll, MulticastDelegateUtilities.SafeInvoke: LengthPrefixCommunicationChannel: MessageReceived: Invoking callback 1/1 for Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.TestRequestHandler.OnMessageReceived, took 1 ms.
TpTrace Information: 0 : 11700, 9, 2023/04/18, 15:15:28.449, 4229278199092, testhost.dll, SocketClient.PrivateStop: Stop communication from server endpoint: 127.0.0.1:062678, error:
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:28.449, 4229278201104, testhost.dll, LengthPrefixCommunicationChannel.Dispose: Dispose reader and writer.
TpTrace Verbose: 0 : 11700, 9, 2023/04/18, 15:15:28.449, 4229278201727, testhost.dll, TcpClientExtensions.MessageLoopAsync: exiting MessageLoopAsync remoteEndPoint: [::ffff:127.0.0.1]:62678 localEndPoint: [::ffff:127.0.0.1]:62679
